zenoblend/main.cpp

```diff
@@ -338,20 +338,21 @@ PYBIND11_MODULE(pylib_zenoblend, m) {
         auto const &attr = mesh->loop.attrs.at(attrName);
         auto attrIndex = attr.index();
         auto loopColor = reinterpret_cast<MLoopCol *>(loopColorPtr);
+        const double gamma = 1.0 / 2.2;
         if (attrIndex == 0) {
             #pragma omp parallel for
             for (int i = 0; i < loopCount; i++) {
                 auto color = std::get<0>(attr)[i];
-                loopColor[i].r = static_cast<unsigned char>(zeno::clamp(color[0] * 255, 0, 255));
-                loopColor[i].g = static_cast<unsigned char>(zeno::clamp(color[1] * 255, 0, 255));
-                loopColor[i].b = static_cast<unsigned char>(zeno::clamp(color[2] * 255, 0, 255));
+                loopColor[i].r = static_cast<unsigned char>(zeno::clamp(pow(color[0], gamma) * 255.0, 0.0, 255.0));
+                loopColor[i].g = static_cast<unsigned char>(zeno::clamp(pow(color[1], gamma) * 255.0, 0.0, 255.0));
+                loopColor[i].b = static_cast<unsigned char>(zeno::clamp(pow(color[2], gamma) * 255.0, 0.0, 255.0));
```

Comment:
Actually Blender's color management already do srgb->rgb in my test.. so I removed this, am I wrong?

Reply:
I guess you could check that with blender spreadsheet as I shown above. For example, if `clr`=0.2 and `Zeno_clr`=0.195, I believe we get the correct result, though precision loss will occur when converting float into char.

Reply:
IMO since 8-bit `char` has limited precision, it's sensible to get color non-linearly encoded whenever `char` is used to represent color. That will reserve more information that human eyes can recognize.